### ISSUE TITLE
Create BFT voting for adjust child epoch length

### ISSUE THREAD

**Issue by mitchelli** [aeternity/aeternity#4428]
Implement BFT voting as described in the [whitepaper](https://github.com/aeternity/hyperchains-whitepaper/blob/master/Periodically-Syncing-HyperChains.md#bft-voting-process)

**Comment by hanssv** [comment]
The voting should also be about which fork to follow, etc. But the BFT part should be the same regardless 😅

**Comment by mitchelli** [comment]
For the first version, I was intending just to set the child epoch length to finish off the PC/CC speed adjustment. And because I thought setting the child epoch length was the easiest part for me to implement. 😊

**Comment by hanssv** [comment]
Setting the value should be the easy part - the voting is harder.

**Comment by mitchelli** [comment]
Yes, it seems much harder but I didn't want the PR to take too long.

**Comment by CanutoIX** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
